src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -261,6 +265,19 @@ public void runReplay(ReplayBean item) {
     }
   }
 
+  public void runRehost(ReplayBean item) {
```

Comment:
runRehost doesn't make sense in this context as it is not running a file. I would do more hostFromReplay